client/src/actions/authActions.js

```diff
@@ -1,9 +1,22 @@
 import { SET_CURRENT_USER } from './types';
 
-// `userTokenData` includes the username + token expiration time
-export const setCurrentUser = userTokenData => ({
-  type: SET_CURRENT_USER,
-  payload: userTokenData
-});
+// userData = { username: string, expires: string }
+export const loginUser = userData => (dispatch) => {
```

Comment:
Object to store in localStorage: `{ username: string, expires: string }`
Because it's an object, we need to stringify it before we can store in localStorage.